app/src/main/res/drawable/text_btn_background_.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
DID YO DESIGNER TELL YOU TO DO THIS BOI

Reply:
this is purely for understanding the visual box of the buttons for testing, this can be removed if deemed needed to